project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * The package that currently holds all the java files of the food-mood project.
+ */
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+import static com.google.appengine.repackaged.
```

Comment:
Is this in two lines because of the linter? (should be in one line without a problem)

Reply:
done